src/main/resources/templates/macros/json/output.vm

```diff
@@ -1,6 +1,6 @@
-#macro(includeOutput, $outputs, $isPassed)
+#macro(includeOutput $outputs, $isPassed)
 
-#if($outputs)
+#if($outputs.size() > 0)
```

Comment:
The empty array is 'truthy' but does not result in any nested html being embedded.